app/models/user.rb

```diff
@@ -114,6 +115,10 @@ def merge_list_search(string)
       search_param = "%#{sanitize_sql_like(string)}%"
       where(search_query, search: search_param)
     end
+
+    def secure_password?(pwd)
+      PASSWORD_PATTERN === pwd
```

Comment:
Hmmm I've never seen `===` used to check regex. I normally would use
`pwd.match(PASSWORD_PATTERN)`

Reply:
It's a case equality that `case` uses under the hood.
https://ruby-doc.org/core-3.0.2/Regexp.html#method-i-3D-3D-3D
Do you want me to use `#string.match`?

Reply:
DONE